packages/soundux/soundux.pacscript

```diff
@@ -1,13 +1,17 @@
 name="soundux"
 version="0.2.7"
-url="https://github.com/Soundux/Soundux/releases/download/"$version"/soundux-"$version".tar.gz"
```

Comment:
Why remove the `"`? It prevents word globbing and other weird things..

Reply:
It's already in quotes

Reply:
Hmm... ok then.